test/e2e/apimachinery/watch.go

```diff
@@ -21,7 +21,7 @@ import (
 	"math/rand"
 	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Why?

Reply:
I recall a change in a recent version of `goimports` that started doing this. Can't say for sure if that's the culprit here...

Reply:
Huh, I'll check my tooling.. Reverting this out of this PR now.